packages/libs/error-stack/src/lib.rs

```diff
@@ -399,25 +407,30 @@
 //!  Feature   | Description                                                    | implies | default
 //! -----------|----------------------------------------------------------------|---------|--------
 //!  `std`     | Enables support for [`Error`] and, on nightly, [`Backtrace`]   |         | enabled
-//!  `hooks`   |Enables the usage of [`set_display_hook`] and [`set_debug_hook`]| `std`   | disabled
 //! `spantrace`| Enables the capturing of [`SpanTrace`]s                        |         | disabled
 //!  `futures` | Provides a [`FutureExt`] adaptor                               |         | disabled
 //! `small`    | Enable optimizations for the memory footprint of [`Report`]    |         | enabled
 //!  `anyhow`  | Provides conversion from [`anyhow::Error`] to [`Report`]       |         | disabled
 //!   `eyre`   | Provides conversion from [`eyre::Report`] to [`Report`]        |         | disabled
+//!   `glyph`  | Provide color and use of box drawing characters in [`Debug`] output |    | enabled
+//! `unstable` | Enables unstable features, which do not follow semver[^unstable] |  | disabled
+//!
+//! [^unstable]: unstable features may be removed in **any** future version without notice.
+//!     They exist to gauge interest towards features that may be stablized in the future.
 //!
 //! [`set_display_hook`]: Report::set_display_hook
 //! [`set_debug_hook`]: Report::set_debug_hook
 //!
 //! [`Error`]: std::error::Error
 //! [`Error::provide`]: std::error::Error::provide
 //! [`Backtrace`]: std::backtrace::Backtrace
-//! [`Display`]: std::fmt::Display
+//! [`Display`]: core::fmt::Display
+//! [`Debug`]: core::fmt::Debug
 //! [`SpanTrace`]: tracing_error::SpanTrace
 //! [`smallvec`]: https://docs.rs/smallvec
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(nightly, feature(provide_any))]
-#![cfg_attr(all(doc, nightly), feature(doc_auto_cfg))]
+#![cfg_attr(all(doc, nightly), feature(doc_auto_cfg, doc_cfg))]
```

Comment:
As we mark `pub mod fmt` as private on `no-std` we don't need this anymore.
```suggestion
#![cfg_attr(all(doc, nightly), feature(doc_auto_cfg))]
```